dashboard/app/helpers/application_helper.rb

```diff
@@ -65,6 +65,8 @@ def best_activity_css_class(user_levels)
       'review_rejected'
     elsif result == Activity::REVIEW_ACCEPTED_RESULT
       'review_accepted'
+    elsif user_level.try(:locked)
+      'locked'
```

Comment:
The precedence shouldn't matter as these two features should be mutually exclusive.